tests/CarbonPeriod/ToDatePeriodTest.php

```diff
@@ -22,7 +23,7 @@ public function testToArrayIsNotEmptyArray()
         $result = $period->toDatePeriod();
 
         $this->assertFalse($period->isEndExcluded());
-        $this->assertSame('DatePeriod', get_class($result));
+        $this->assertInstanceOf(DatePeriod::class, $result);
```

Comment:
This test is still broken. `assertInstanceOf` is not what this test aims to verify.